src/components/Footer/style.css

```diff
@@ -35,20 +35,18 @@ footer{
     margin-bottom: 20px;
     flex: 1;
 }
-.link-advertising{
+.additional-link--img{
```

Comment:
1. This should be styles of  ListLinks.
2. I wonder what `--img` means... Why is it separated with `--`? Did you do this with any methodology in mind?

Reply:
fixed

Reply:
I don't understand how you fixed it.

Do you use BEM methodology? If yes, then what does this `--img` mean?

From BEM Quick Start `block-name__element-name_modifier-name`.

https://en.bem.info/methodology/quick-start/#modifier